src/core/public/application/application_service.tsx

```diff
@@ -117,6 +118,11 @@ export class ApplicationService {
       return replace ? this.history!.replace(url, state) : this.history!.push(url, state);
     };
 
+    this.navigateToNewTab = (url) => {
+      // window.open copies session information if base url is same
+      return window.open(basename + url, '_blank');
```

Comment:
Is that true for all supported browser? My stack overflow skillz seems to provide mixed answers (https://stackoverflow.com/questions/20879714/how-to-prevent-sessionstorage-being-inherited-when-using-target-blank-window), some seems to share, not copy, the storage.

Reply:
yes share is a better word, i have tested in firefox, chrome and safari and it seems to work pretty well and from the web, i don't think there are any browser support concerns